### PR TITLE
[ISSUE-49] generate nginx config string

### PR DESCRIPTION
**Pull request by goromlagche**

### CODE REVIEW

File: apps/dockup/lib/dockup/nginx_config.ex [hunk 1]
```diff
@@ -42,4 +51,18 @@ defmodule Dockup.NginxConfig do
     File.write(config_file(project_id), config)
     %{"80" => url}
   end
+
+  def proxy_passing_port({a,b}) do
```
Comment (emilsoman) [inline apps/dockup/lib/dockup/nginx_config.ex]: Let's use better variable names

**Comment by emilsoman** [comment]
@goromlagche can you add a test for this ?

File: apps/dockup/lib/dockup/nginx_config.ex (hunk 1 above)
Comment (goromlagche) [inline apps/dockup/lib/dockup/nginx_config.ex]: done, please check.

**Comment by goromlagche** [comment]
@emilsoman added specs https://github.com/code-mancers/dockup/pull/52/commits/5deae7eeb1ac3c6ce8edcce468c674a20fc51df3 , please check

**Comment by emilsoman** [comment]
Nice code 👍

**Pull request merged**